e2e/cypress/integration/login/tutorial_navigation_and_links_spec.js

```diff
@@ -41,13 +41,7 @@ describe('Test Tutorial Navigation', () => {
         checkStepTwo();
 
         // * Test the App Download Image link to ensure it returns a 200 result.
-        cy.get('#appDownloadImage').then(($a) => {
-            const href = $a.prop('href');
-
-            cy.request(href).then((response) => {
-                expect(response.status).to.eq(200);
-            });
-        });
+        cy.request('href').its('status').should('equal', 200);
```

Comment:
`'href'` should be the actual URL we want to request, no just a string of 'href'.

```js
cy.get('#appDownloadImage').should('have.attr', 'href').then((href) => {
       cy.request(href).its('status').should('equal', 200)
})
```

Reply:
Fixed.  Weird that it didn't error.